apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -196,15 +191,8 @@ class SectionProgress extends Component {
               className="fa-pulse fa-3x"
             />
           )}
-          {levelDataInitialized && currentView === ViewType.SUMMARY && (
-            <div id="uitest-summary-view" style={summaryStyle}>
-              <ProgressTableSummaryView />
-            </div>
-          )}
-          {levelDataInitialized && currentView === ViewType.DETAIL && (
-            <div id="uitest-detail-view" style={detailStyle}>
-              <ProgressTableDetailView />
-            </div>
+          {levelDataInitialized && (
+            <ProgressTableContainer currentView={currentView} />
```

Comment:
don't we need to make sure `currentView !== ViewType.STANDARDS`? or return null from container's render in that case?

Reply:
😱  I was using a CSD course and didn't take into account the standards view! Excellent catch! I will update and add a test